### DISCUSSION TITLE
Enabling percolation from the deepest soil layer

### DISCUSSION THREAD

**Discussion by calvarezgarreton** [mhm-ufz/mHM#64] [Q&A]
Dear mHM team,
We are evaluating mHM simulations over a basin in Chile (3-km) based on soil moisture measurements (point measurement) and realized that the first 2 soil layers were very well represented by mHM. However, the soil layers below 15 cm are simulated as fully saturated at all times, in contrast to the observations on those depths. We think this happens because the deepest layer can not percolate and thus the soil storages can not drain from below. If we allow a water flux to leave the deepest soil layer into a deep aquifer, the model might be able to represent a more realistic water content. 
Is there a way this flux can be activated in the current version of mHM?
Thanks,
Camila

**Comment by rohinikumar82** [comment]
Dear Camila,

In mHM - the percolation fluxes from the lowest soil layer are first passed to the unsaturated zone and then from there water recharge to the deeper GW (aquifer) zone.  Percolation flux from soil layers is always activated in mHM. 

To your specific case, maybe you would consider checking how the soil parameters for the lowest soil layer look like. This could better give the idea if those parameters, responsible for SM, ET and percolation dynamics, can be adjusted before you make changes/adjustments to the model code. Also possibly check the dynamics of the percolation fluxes. 

Best regards,
Rohini
